assets/scss/seasons/3_harryette_mullen.scss

```diff
@@ -0,0 +1,63 @@
+.mullen {
+  padding-bottom: initial;
+  background-color: #224fc7;
+  color: white;
+  font-family: 'Times';
+  align-items: center;
+  margin-top: var(--margin);
+
+
+
+  .layout-wrapper {
+    display: flex;
+    width: 462px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    margin: auto;
+
+
+    @include mobile {
+      width: 415px;
+      margin: auto;
+    }
+  }
+
+
+  .poem {
+    font-size: var(--font-size);
+    padding-bottom: 60px;
```

Comment:
i would use padding in multiples of the margin so that there is some repeated dimensions.
```
padding-bottom: calc(2 * var(--margin));
```
for example